bool enable

---

bool cook_done